extensions/CommentingControllerTwitterUsernameExtension.php

```diff
@@ -0,0 +1,11 @@
+<?php 
+
+class CommentingControllerTwitterUsernameExtension extends DataExtension {
```

Comment:
Possibly though it does need a check in the template as to whether or not the Twitter username is part of the comment during rendering.  Hence why I included it, but  left it optional to enable it.